src/index.ts

```diff
@@ -82,6 +140,74 @@ const main = (vortex: VortexExtensionContext) => {
     wrapInstall(vortex, vortexApi, internalPipelineInstaller, CurrentFeatureSet),
   );
 
+  if (CurrentFeatureSet.REDmodLoadOrder === Feature.Enabled) {
+    vortex.registerLoadOrder({
+      gameId: GAME_ID,
+
+      // This needs to be actually implemented, it doesnt't do
+      // anything on its own, so leave it out now to avoid confusion
+      //
+      // toggleableEntries: true,
+
+      // Can add instructions to the right-hand panel. Might be useful,
+      // but on the whole I think it's probably better to reduce the
+      // amount of space the panel takes instead.
+      //
+      usageInstructions: heredoc(`
+        You don't have to order everything. It's best to only order mods that
+        require it, or that you otherwise know to conflict with each other.
+        [br][/br]
+        Only REDmods and autoconverted heritage mods are orderable. If you don't see
+        something you just installed, click on Refresh.
+        [br][/br]
+        You can order both enabled and disabled mods, but only the enabled ones will
+        be included in the REDmod deployment. The disabled ones will remember their
+        place in the load order, though, so long as you don't uninstall them!
+        [br][/br]
+        All heritage archive mods that are not autoconverted to REDmod will be loaded
+        AFTER all REDmods, in the usual alphabetical order.
+        [br][/br]
+        REDmods that you have installed outside Vortex are NOT supported right now.
+        [br][/br]
+        The load order is saved automatically, and will be deployed whenever the next
+        Vortex deployment occurs - you can also manually click to deploy, if you like!
+        [br][/br]
+        REDmod deployment can take a little while if you have tweak or script mods,
+        so wait for the green success notification before you start the game! :)
+        [br][/br]
+        You can still use the redmod tool manually, too, but changes won't be reflected
+        in Vortex.
+      `),
```

Comment:
I like this. It's good to have a nice explanation. Hopefully it mitigates the issues that I see happening... assuming people read.